### ISSUE TITLE
Barb Meditations

### ISSUE THREAD

**Issue by RyanEOD** [rpherbig/dr-scripts#3171]
In base-spells.yaml the Barb Meditations have their activated messages of when you start the meditation. Without the Yogi Mastery, it still takes a few seconds to complete after the round time before you can stand up. So, can we change the Meditations from;

`activated_message: ^You .* to meditate`

to

`activated_message: feel a jolt as your vision snaps shut`

I've edited them all in the attached base-spells.yaml but I'm not quite sure how to get it added to the "base" so my file doesn't keep getting updated locally.
[base-spells.zip](https://github.com/rpherbig/dr-scripts/files/2454006/base-spells.zip)

Thanks!

**Comment by RyanEOD** [comment]
@rpherbig  - Sure, I've tested it and I don't have Yogi, also given it to a few others without Yogi to use. I'll see if i can scrounge up someone with Yogi to test it.

**Comment by MahtraDR** [comment]
Works with Yogi.

``` --- Lich: buff active.
 [buff]>meditate focus                                                                                                                                                                                                                        
 You begin to meditate upon the chakrel amulet, your inner fire swelling as you center your mind, body, and spirit.                                                                                                                           
 Roundtime: 8 sec.                                                                                                                                                                                                                            
 R>                                                                                                                                                                                                                                           
 Peering into the flame, you visualize the difficult feats you wish to perform.                                                                                                                                                               
 R>                                                                                                                                                                                                                                           
 You draw the flames close and bring forth an image of the complex task of disabling locks and traps.                                                                                                                                         
 R>                                                                                                                                                                                                                                           
 The black ape begins to advance on you!                                                                                                                                                                                                      
 The black ape is still a distance away from you and is closing steadily.                                                                                                                                                                     
 >                                                                                                                                                                                                                                            
 The flame surrounds you and infuses you with deep insight into the workings of locks and traps!                                                                                                                                              
 You feel a jolt as your vision snaps shut.                                                                                                                                                                                                   
 --- Lich: buff has exited. ```

@rpherbig Can we close this one?

**Comment by rpherbig** [comment]
Fixed in #3581
